### ISSUE TITLE
WARNING: Protocol IKVReduce implemented multiple times

### ISSUE THREAD

**Issue by arichiardi** [metosin/spec-tools#143]
Hi again,
with the latest ClojureScript master I get a warning:

```
WARNING: Protocol IKVReduce implemented multiple times at line 285 /home/arichiardi/git/spec-tools/src/spec_tools/core.cljc
```

Just wanted to report it :smile:

**Comment by miikka** [comment]
Nice, it means that https://dev.clojure.org/jira/browse/CLJS-1297 has been fixed.

**Comment by ikitommi** [comment]
Fixed in 0.8.1
